Neuron/Sections/Wallet/View/AddAssetController.swift

```diff
@@ -43,12 +44,14 @@ class AddAssetController: UIViewController, UITableViewDelegate, UITableViewData
             Toast.showToast(text: "Token信息不全，请核对合约地址是否正确")
             return
         }
-        let appModel = WalletRealmTool.getCurrentAppModel()
-        if appModel.extraTokenList.contains(tokenModel) {
+        let result = tokenArray.filter { (token) -> Bool in
```

Comment:
We are checking if `tokenArray` already has the token. `contains(where:)` should be used instead of `filter`.